### YamlMime:ManagedReference
items:
- uid: System.Threading.ThreadInterruptedException
  id: ThreadInterruptedException
  children:
  - System.Threading.ThreadInterruptedException.#ctor
  - System.Threading.ThreadInterruptedException.#ctor(System.String)
  - System.Threading.ThreadInterruptedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Threading.ThreadInterruptedException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ThreadInterruptedException
  nameWithType: ThreadInterruptedException
  fullName: System.Threading.ThreadInterruptedException
  type: Class
  summary: "Wyjątek zgłaszany, gdy <xref href=&quot;System.Threading.Thread&quot;> </xref> zostało przerwane, gdy jest on w stanie oczekiwania."
  remarks: "Po utworzeniu wątek jest w co najmniej jednej <xref:System.Threading.ThreadState>stwierdza, dopóki nie zostanie on zniszczony.</xref:System.Threading.ThreadState> Wywoływanie <xref:System.Threading.Thread.Interrupt%2A>gdy wątek jest w <xref:System.Threading.ThreadState>stanu spowoduje, że threadinterruptedexception — zostanie wygenerowany w wątku docelowych.</xref:System.Threading.ThreadState> </xref:System.Threading.Thread.Interrupt%2A> Jeśli wątek nie znajduje się w <xref:System.Threading.ThreadState>stanu nie jest zwracany wyjątek, aż wątek przechodzi tego stanu.</xref:System.Threading.ThreadState> Jeśli wątek nigdy nie blokuje, można wykonać bez kiedykolwiek przerwane.       Threadinterruptedexception — używa COR_E_THREADINTERRUPTED HRESULT, która ma wartość 0x80131519.       Aby uzyskać listę początkowe wartości właściwości dla wystąpienia threadinterruptedexception — zobacz <xref:System.Threading.ThreadInterruptedException.%23ctor%2A>konstruktorów.</xref:System.Threading.ThreadInterruptedException.%23ctor%2A>"
  example:
  - "The following code example shows the behavior of a running thread when it is interrupted and subsequently gets blocked.  \n  \n [!code-cpp[System.Threading.Thread.Interrupt#1](~/add/codesnippet/cpp/t-system.threading.threa_1_1.cpp)]\n [!code-vb[System.Threading.Thread.Interrupt#1](~/add/codesnippet/visualbasic/t-system.threading.threa_1_1.vb)]\n [!code-cs[System.Threading.Thread.Interrupt#1](~/add/codesnippet/csharp/t-system.threading.threa_1_1.cs)]"
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class ThreadInterruptedException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Threading.ThreadInterruptedException.#ctor
  id: '#ctor'
  parent: System.Threading.ThreadInterruptedException
  langs:
  - csharp
  name: ThreadInterruptedException()
  nameWithType: ThreadInterruptedException.ThreadInterruptedException()
  fullName: System.Threading.ThreadInterruptedException.ThreadInterruptedException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Threading
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Threading.ThreadInterruptedException&quot;> </xref> klasy przy użyciu domyślnej właściwości."
  remarks: "W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Threading.ThreadInterruptedException>.</xref:System.Threading.ThreadInterruptedException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Ciąg z komunikatem o zlokalizowane. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ThreadInterruptedException ();
    parameters: []
  overload: System.Threading.ThreadInterruptedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Threading.ThreadInterruptedException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Threading.ThreadInterruptedException
  langs:
  - csharp
  name: ThreadInterruptedException(String)
  nameWithType: ThreadInterruptedException.ThreadInterruptedException(String)
  fullName: System.Threading.ThreadInterruptedException.ThreadInterruptedException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Threading
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Threading.ThreadInterruptedException&quot;> </xref> klasy z powodu określonego błędu."
  remarks: "W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Threading.ThreadInterruptedException>.</xref:System.Threading.ThreadInterruptedException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Ciąg z komunikatem o. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ThreadInterruptedException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Komunikat o błędzie opisujący przyczynę wyjątek."
  overload: System.Threading.ThreadInterruptedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Threading.ThreadInterruptedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Threading.ThreadInterruptedException
  langs:
  - csharp
  name: ThreadInterruptedException(SerializationInfo,StreamingContext)
  nameWithType: ThreadInterruptedException.ThreadInterruptedException(SerializationInfo,StreamingContext)
  fullName: System.Threading.ThreadInterruptedException.ThreadInterruptedException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Threading
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Threading.ThreadInterruptedException&quot;> </xref> z zserializowanymi danymi."
  remarks: "Ten konstruktor jest wywoływana podczas deserializacji do odtworzenia obiekt wyjątku przesyłane przez strumień. Aby uzyskać więcej informacji, zobacz [XML i serializacji SOAP](~/add/includes/ajax-current-ext-md.md)."
  syntax:
    content: protected ThreadInterruptedException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "<xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> Zawierający dane zserializowanego obiektu dotyczące zgłaszanego wyjątku."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "<xref:System.Runtime.Serialization.StreamingContext>Zawierający informacje kontekstowe dotyczące źródła lub miejsca docelowego.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.Threading.ThreadInterruptedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Threading.ThreadInterruptedException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Threading.ThreadInterruptedException
  langs:
  - csharp
  name: ThreadInterruptedException(String,Exception)
  nameWithType: ThreadInterruptedException.ThreadInterruptedException(String,Exception)
  fullName: System.Threading.ThreadInterruptedException.ThreadInterruptedException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Threading
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Threading.ThreadInterruptedException&quot;> </xref> z określonego komunikatu o błędzie i odwołania do wyjątek wewnętrzny będący przyczyną tego wyjątku."
  remarks: "Wyjątek zgłaszany bezpośrednio w wyniku poprzedniego wyjątku powinien zawierać odwołanie do poprzedniego wyjątku we <xref:System.Exception.InnerException%2A>Właściwości.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>Właściwość zwraca tę samą wartość, która została przekazana do konstruktora, lub `null` Jeśli <xref:System.Exception.InnerException%2A>Właściwości nie dostarcza do konstruktora wartości wyjątku wewnętrznego.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Threading.ThreadInterruptedException>.</xref:System.Threading.ThreadInterruptedException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Wyjątek wewnętrzny odwołania. |   | <xref:System.Exception.Message%2A>| Ciąg z komunikatem o. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ThreadInterruptedException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Komunikat o błędzie opisujący przyczynę wyjątek."
    - id: innerException
      type: System.Exception
      description: "Wyjątek będący przyczyną bieżącego wyjątku. Jeśli `innerException` parametr nie jest <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, bieżący wyjątek jest zgłaszany w <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloku, który obsługuje wyjątek wewnętrzny."
  overload: System.Threading.ThreadInterruptedException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Threading.ThreadInterruptedException.#ctor
  parent: System.Threading.ThreadInterruptedException
  isExternal: false
  name: ThreadInterruptedException()
  nameWithType: ThreadInterruptedException.ThreadInterruptedException()
  fullName: System.Threading.ThreadInterruptedException.ThreadInterruptedException()
- uid: System.Threading.ThreadInterruptedException.#ctor(System.String)
  parent: System.Threading.ThreadInterruptedException
  isExternal: false
  name: ThreadInterruptedException(String)
  nameWithType: ThreadInterruptedException.ThreadInterruptedException(String)
  fullName: System.Threading.ThreadInterruptedException.ThreadInterruptedException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Threading.ThreadInterruptedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Threading.ThreadInterruptedException
  isExternal: false
  name: ThreadInterruptedException(SerializationInfo,StreamingContext)
  nameWithType: ThreadInterruptedException.ThreadInterruptedException(SerializationInfo,StreamingContext)
  fullName: System.Threading.ThreadInterruptedException.ThreadInterruptedException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Threading.ThreadInterruptedException.#ctor(System.String,System.Exception)
  parent: System.Threading.ThreadInterruptedException
  isExternal: false
  name: ThreadInterruptedException(String,Exception)
  nameWithType: ThreadInterruptedException.ThreadInterruptedException(String,Exception)
  fullName: System.Threading.ThreadInterruptedException.ThreadInterruptedException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Threading.ThreadInterruptedException.#ctor*
  parent: System.Threading.ThreadInterruptedException
  isExternal: false
  name: ThreadInterruptedException
  nameWithType: ThreadInterruptedException.ThreadInterruptedException
